src/test/java/seedu/address/logic/commands/FindPropertyCommandTest.java

```diff
@@ -227,19 +291,88 @@ public void multiOptionTest() {
     /**
      * Parses {@code userInput} into a {@code Predicate}.
      */
-    private PropertyPredicateList preparePredicate(String userInput) {
-        String[] nameKeywords = userInput.split("\\s+");
+    private PropertyPredicateList preparePredicate(String args) throws ParseException {
```

Comment:
I dont think its a good idea to duplicate the code from `FindPropertyCommand#parse` to here again. Given that this is a test class, is it possible to manually create the `Predicate`/`PredicateList` in each of the test itself instead?